### PR TITLE
Add new quest tab for UEV tier

### PR DESCRIPTION
**Pull request by NotAPenguin0**
This PR attempts to lift some of the quests out of the endgame goals tab into its own tab for UEV, as well as add some additional quests to guide the player through the tier.

### CODE REVIEW

**Comment by NotAPenguin0** [comment]
image
image

Images of current versions of UEV and endgame goals tabs

File: config/betterquesting/DefaultQuests/Quests/Tier10UEV-D_aFb1hFQH2PDkN0_KvAEQ==/ADigitalSingular-L58SFyBoRj-Lc4Iqu8WuHA==.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    "0:10": {
+      "partialMatch:1": 1,
+      "autoConsume:1": 0,
+      "groupDetect:1": 0,
+      "ignoreNBT:1": 1,
+      "index:3": 0,
+      "consume:1": 0,
+      "requiredItems:9": {
+        "0:10": {
+          "id:8": "appliedenergistics2:item.ItemExtremeStorageCell.Singularity",
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/Tier10UEV-D_aFb1hFQH2PDkN0_KvAEQ==/ADigitalSingular-L58SFyBoRj-Lc4Iqu8WuHA==.json:46]: they already have quests. no multiple quests for the same thing please.

**Review by chochem — DISMISSED** [review]
see comment

**Review by chochem — APPROVED** [review]
thank you! looks all fine to me.

**Pull request merged**